migrations/20231018140903-create-todo.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Todos', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      title: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      description: {
+        type: Sequelize.TEXT,
+        allowNull: false
+      },
+      status: {
+        type: Sequelize.ENUM,
+        allowNull: false,
+        values: ["created", "in_progress", "done"]
+      },
```

Comment:
looks good to me semua sebenernya, 
paling kalau diminta untuk kasih tambahan, untuk sesuatu yang valuenya udah pasti kaya enum gini, dan kamu udah tau status paling awal apa bisa ditambahin defaultValue, diluar hooks yang udah dikerjain, overall goods,